### DISCUSSION TITLE
.save files in config directory

### DISCUSSION THREAD

**Discussion by tinkerpunktheprol** [i3/i3#4686] [Q&A] [answered]
Hello, I was wondering if anyone else has these extra files in their config directory. I have three of them;

`i3blocks.conf.save`
`config.save`
`config.save.1`

I am guessing they are some sort of auto-save files that are created when I exit without saving, or maybe when I mess the config file up and get an error. I'm just wondering if they are safe to delete, and what I can do to prevent them from being created in the future. 

Strangely I wasn't able to find anything online about this.

OS: Endeavour OS

DE/WM: i3wm (I do have KDE & KDE+i3 installed as well, but I only use i3 by itself)

Editor: Emacs

**Comment by Airblader** [comment, marked as answer]
It does look like files created by your editor, but you didn't mention what editor you're using.

In that case they should be safe to delete. An easy way to try out is to move them elsewhere instead and having a backup of your config.

**Reply by tinkerpunktheprol** [reply]
Sorry, that's a good point. I am using Emacs. I'll edit the post as well.

**Reply by tinkerpunktheprol** [reply]
It looks like it's an Emacs thing, though I can't find any mention of the .save file extension. That could be a Doom Emacs thing, though. I'd need to dig into the auto-save-mode.el to know for sure, but I'm fairly confident this is what is happening. They usually get deleted, but these are probably a result of a system crash or something. Thanks for your help.

https://www.emacswiki.org/emacs/AutoSave
